pages/common/erl.md

```diff
@@ -0,0 +1,15 @@
+# erl
+
+> Run and manage programs written in Erlang programming language.
+
+- Run sequential erlang program as a common script and then exit:
```

Comment:
Should `erlang` be capitalised here, or should it be lowercase in the description?

Reply:
I think capitalising here is better, going to fix it